Library/Formula/gvp.rb

```diff
@@ -17,11 +15,20 @@ def install
     system "make", "install"
   end
 
+  def caveats; <<-EOS.undent
+    To have gvp modify your environment, add the following at the end of your
+    .bashrc or .zshrc:
+
+      source gvp
```

Comment:
Took a little look at its manual, it seems that `source gvp` is a command invoked by users rather than added in `.bashrc` or `.zshrc`.